docs/changelog.md

```diff
@@ -11,6 +11,10 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
 
+* **support-test**
+  * ⚠️ **This is a breaking change**: `MainCoroutineRule` constructor now takes a `TestDispatcher` instead of deprecated `TestCoroutineDispatcher`. Default is `UnconfinedTestDispatcher`.
+  * ⚠️ **This is a breaking change**: `MainCoroutineRule.runBlockingTest` is replaced with a `runTestOnMain` top level function. . This method is preferred over the global `runTest` because it reparents new child coroutines to the test coroutine context.
```

Comment:
Heads-up that this is used in Fenix: https://github.com/mozilla-mobile/fenix/blob/main/app/src/test/java/org/mozilla/fenix/settings/deletebrowsingdata/DeleteAndQuitTest.kt#L14 and will need a change.

Reply:
The matching Fenix changes are also ready for review and [the `runBlockingTest` usages there was indeed replaced with `runTestOnMain`](https://github.com/mozilla-mobile/fenix/pull/24892/files#diff-8c0a90dde94db153441c370ee85c29abe8402ba68915efad87a3954af5dfec6c) though they were already marked as ignored. And I kept them as such to not pollute my potential failing tests with other unrelated failures.